### PR TITLE
docs: updates the discuss moderators list

### PR DESCRIPTION
**Pull request by raelga**
Updates the moderator list for the Discuss platform.

/sig contributor-experience
/area community-management
/cc @mrbobbytables
/cc @cartyc

### CODE REVIEW

File: communication/moderators.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ Administrators seats: 4
 
 | Name          | Kubernetes Discuss ID | Region   | Timezone                                                |
 | ------------- | --------------------- | -------- | ------------------------------------------------------- |
-| Jorge Castro  | @castrojo             | Americas | [ET - Eastern Time (US East Coast)](https://time.is/ET) |
```
Comment (mrbobbytables) [inline communication/moderators.md:85]: castrojo is still an admin, it's alejandrox1 that has been removed. I should have been more clear as there are two jorges^^;;;;;

Comment (raelga) [reply communication/moderators.md:85]: Updated and sorted alphabetically.

**Pull request merged**